step1_masking_experiment.py

```diff
@@ -48,19 +48,20 @@
 
```

Comment:
Somehow I cannot comment above, sorry for this mess but this is a way to create different objects with the same input:
```suggestion
detec_txt = local_txt = disc_txt = disc_txt2 = disc_txt3 = ''
```